recipes/metabolisHMM/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Decompress the HMM database
+tar zxvf metabolisHMM_markers_v1.9.tgz
+
+# Install python libraries
+python3 -m pip install .
+
+# Copy scripts to environment bin/
```

Comment:
Hm, is this necessary? The `pip install` step should have copied the binaries/scripts already.

Reply:
I thought so too, but it did not work on my local build